AI/modules/data_collector/ticker_master_updater.py

```
@@ -0,0 +1,196 @@
+            company_name_data = [
+                (item['ticker'], item['name']) 
+                for item in ticker_list 
+                if item.get('name')
+            ]
+            if company_name_data:
+                # upsert 로직이 복잡하므로 개별 실행 혹은 executemany (PK 충돌 주의)
+                # 여기서는 단순화를 위해 반복문 사용 (데이터 양이 적으므로)
+                for t, n in company_name_data:
+                    try:
+                        cursor.execute(query_company_names, (t, n))
+                    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)